src/app/features/reporter/reporter-dialog.component.html

```diff
@@ -66,5 +66,5 @@
 </div>
 <systelab-dialog-bottom>
 	<button type="button" class="btn mr-0" (click)="doUpdateTestCase()">Update Test Case</button>
```

Comment:
The button Update Test Case must be disabled when the project field is not informed. Please, add the [disabled]=!this.selectedProjectId